cli/cmd/devnet.go

```diff
@@ -177,7 +177,9 @@ func devnetDefinition(ctx context.Context) (app.Definition, error) {
 		imgTag = "main"
 	}
 
-	defCfg.OmniImgTag = imgTag
+	defCfg.HaloImgTag = imgTag
+	defCfg.RelayerImgTag = imgTag
+	defCfg.MonitorImgTag = imgTag
```

Comment:
lets stick to existing two different patterns:
 - `omniTag` is overrride for all containers, specified in flags.
 - `pinned_*_tag` is specified in manifest only.